pkgs/games/classicube/default.nix

```diff
@@ -0,0 +1,57 @@
+{ stdenv, fetchFromGitHub, curl, libX11, libGL, openal, dos2unix, liberation-sans-narrow, bash }:
+
+stdenv.mkDerivation rec {
+  pname = "ClassiCube";
+  version = "1.0.7";
+
+  src = fetchFromGitHub {
+    owner = "UnknownShadow200";
+    repo = pname;
+    rev = version;
+    sha256 = "1ivxf5ffar9fxnri6xrdk5b77b483hvbjd8d6sifafslpxk3l1r1";
+  };
+
+  nativeBuildInputs = [ dos2unix ];
+
+  postUnpack = ''
+    sourceRoot=$sourceRoot/src
+  '';
+
+  prePatch = ''
+    dos2unix Platform.c
+  '';
+
+  patches = [ ./font-patch.patch ];
+
+  font = "${liberation-sans-narrow}/share/fonts/truetype/LiberationSansNarrow-Regular.ttf";
+
+  postPatch = ''
+    substituteInPlace Platform.c --replace %fontdir% ${font}
+    substituteInPlace Makefile --replace JOBS=1 JOBS=${ if enableParallelBuilding then "$NIX_BUILD_CORES" else "1"}
+  '';
```

Comment:
Can you make this conditional on enableParallelBuilding and replace JOBS with NIX_BUILD_CORES? Also NIX_BUILD_CORES should always be 1 or more I think.

Reply:
No really sure about what you mean. Do you mean using --replace JOBS "" --replace JOBS NIX_BUILD_CORES ?